src/tables.ts

```diff
@@ -821,22 +824,22 @@ export const Tables = {
             // Breve note
             code_head: 'noteheadDoubleWhole',
             getWidth(scale = Tables.DEFAULT_NOTATION_FONT_SCALE): number | undefined {
-              return new Glyph('noteheadDoubleWhole', scale).getMetrics().width;
+              return Glyph.getWidth(Flow.DEFAULT_FONT_STACK, 'noteheadDoubleWhole', scale);
```

Comment:
Here and in the instances below use:
`Tables.DEFAULT_FONT_STACK`
rather than
`Flow.DEFAULT_FONT_STACK`

Reply:
Unfortunately, using `Tables.DEFAULT_FONT_STACK` here doesn't work here because it doesn't get updated if a user assigns a new value to `Flow.DEFAULT_FONT_STACK`. I don't like the circular reference either but I saw that pattern was used elsewhere in the code without apparent problems.

Reply:
I think that it should work as arrays are copied by reference. See https://jsfiddle.net/27s0L1xh/

Reply:
Sorry, I didn't explain myself clearly.

The problem is that in the compiled JS code, `Flow` and `Tables` are different objects (because of how `Flow` is [initialised here](https://github.com/0xfe/vexflow/blob/24f91ee58fa2de3536b56ee90604d2dab36e498c/src/flow.ts#L80-L81)). Assigning a new array to `Flow.DEFAULT_FONT_STACK` cannot possibly affect the value of `Tables.DEFAULT_FONT_STACK`. Consequently, any code that refers to `DEFAULT_FONT_STACK` _must_ use `Flow.DEFAULT_FONT_STACK` rather than `Tables.DEFAULT_FONT_STACK` to function correctly.

The compiled JS code is effectively doing the following:

```javascript
let Tables = { DEFAULT_FONT_STACK: [1] };
let Flow = { DEFAULT_FONT_STACK: Tables.DEFAULT_FONT_STACK };
```

Clearly, assigning a new array to `Flow.DEFAULT_FONT_STACK` cannot affect the value of`Tables.DEFAULT_FONT_STACK`.

Reply:
One more thing I was perhaps not clear about:

The issue I'm referring to is when the a user assigns an entirely new array to `Flow.DEFAULT_FONT_STACK` rather than modifying the existing one in-place. This is an important use case because it's the [documented method](https://github.com/0xfe/vexflow/wiki/VexFlow-Font-Rendering#the-vexflow-font-stack) of modifying the font stack.